Cargo.toml

```diff
@@ -13,7 +13,7 @@ license = "GPL-3.0-or-later"
 [workspace.dependencies]
 kryoptic = { version = "1.2.0", path = ".", default-features = false }
 ossl = { version = "1.2.0", path = "ossl", default-features = false }
-pkcs11 = { version = "1.2.0", path = "pkcs11", default-features = false }
+pkcs11 = { version = "1.2.0", path = "pkcs11", default-features = false, features = ["pkcs11_3_2"] }
```

Comment:
We probably do not need to add the `features = ["pkcs11_3_2"]` here when it was removed?

Reply:
it was not removed in the pkcs11_3_2 crate

Reply:
I think its for no good use even in the pkcs11 crate now when we consider them final so.